pom.xml

```diff
@@ -92,6 +92,10 @@
 
     <version.org.drools.guvnor-api>5.6.0.Final</version.org.drools.guvnor-api>
     <version.org.jboss.errai>4.0.1-SNAPSHOT</version.org.jboss.errai>
+
+    <!-- Required for kie-wb-common to build in IntelliJ Idea 2017 -->
+    <version.org.jboss.errai.wildfly>10.0.0.Final</version.org.jboss.errai.wildfly>
```

Comment:
Should be "version.org.wildfly" instead and let's use the latest and greatest:

    <version.org.wildfly>10.1.0.Final</version.org.wildfly>

optaplanner-webexamples already has this, but it should be removed there once this PR is merged.